keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -945,27 +873,25 @@ Several components must be implemented or modified in Kubernetes:
   resource is allocated before the Pod gets scheduled, similar to
   https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/scheduling/scheduler_binder.go
 - Kubelet must be extended to manage ResourceClaims
-  and to call a resource kubelet plugin. That plugin returns CDI device ID(s)
+  and to call a kubelet plugin. That plugin returns CDI device ID(s)
   which then must be passed to the container runtime.
 
-A resource driver can have the following components:
-- *CRD controller* (optional): a central component which translates parameters
-  defined with a vendor CRD into in-tree parameter types. 
-- *kubelet plugin* (required): a component which cooperates with kubelet to
-  publish resource information and to prepare the usage of the resource on a node.
+A DRA driver can have the following components:
+- *admission webhook* (optional): a central component which checks the opaque
```

Comment:
We could also provide a sample for this in DRA sample repo.

Reply:
I created https://github.com/kubernetes-sigs/dra-example-driver/issues/49